src/handlers/check_commits/canonicalize_issue_links.rs

```diff
@@ -0,0 +1,56 @@
+use crate::{config::CanonicalizeIssueLinksConfig, github::GithubCommit};
+
+pub(super) fn issue_links_in_commits(
+    _conf: &CanonicalizeIssueLinksConfig,
```

Comment:
Is the config here as a type system proof that the functionality was enabled?

Reply:
Yes, it is. It's a for future-proofing, in case we add configuration options.